test/negative/llvm.fma.ll

```diff
@@ -4,7 +4,8 @@
 ; RUN: llvm-as %s -o %t.bc
 ; RUN: not llvm-spirv %t.bc 2>&1 | FileCheck %s
 
-; CHECK: InvalidFunctionCall: Unexpected llvm intrinsic: llvm.fma.f32
+; CHECK: InvalidFunctionCall: Unexpected llvm intrinsic:
```

Comment:
I kind of like one-liners more. I probably see logic to move intrinsic name to the next line (to prevent double ':' nesting in a single line), but I think while it fits into 80 symbols limitation - there is no need to split the string.

Reply:
This was done not because of too long line, but because of changes in the translator output: it now emits `\n` between this phrase and actual intrinsic name: the next check it not just `CHECK`, but `CHECK-NEXT`